airflow/gcp/operators/cloud_storage_transfer_service.py

```diff
@@ -218,17 +218,17 @@ def _validate_inputs(self):
 
     def execute(self, context):
         TransferJobPreprocessor(body=self.body, aws_conn_id=self.aws_conn_id).process_body()
-        hook = GCPTransferServiceHook(api_version=self.api_version, gcp_conn_id=self.gcp_conn_id)
+        hook = CloudDataTransferServiceHook(api_version=self.api_version, gcp_conn_id=self.gcp_conn_id)
         return hook.create_transfer_job(body=self.body)
 
 
-class GcpTransferServiceJobUpdateOperator(BaseOperator):
+class CloudDataTransferServiceUpdateJobOperator(BaseOperator):
     """
     Updates a transfer job that runs periodically.
 
     .. seealso::
         For more information on how to use this operator, take a look at the guide:
-        :ref:`howto/operator:GcpTransferServiceJobUpdateOperator`
+        :ref:`howto/operator:CloudDataTransferServiceUpdateJobOperator`
```

Comment:
Have you also changed the docs?

Reply:
Commit with all fixes is on the way. I just finished fixing the PR.